app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/_menu.less

```diff
@@ -273,7 +273,16 @@
             > .submenu {
                 transform: translateX(0);
                 visibility: visible;
-                z-index: @submenu__z-index;
+                z-index: 698;
+                overflow-x: scroll;
+                min-height: 98.65%;
+                left: 0;
+                position: absolute;
+                left: 100%;
+                min-width: 100%;
+                display: block;
+                float: left;
+                max-width: 1640px;
```

Comment:
According to [Magento frontend dev guide](https://devdocs.magento.com/guides/v2.0/frontend-dev-guide/responsive-web-design/rwd_css.html) we haven't breakpoint on 1640 window width. Please change it to 1440px